toc/ROLES.md

```diff
@@ -88,10 +88,9 @@ establish sub-working groups. Working groups delegate change approval to Approve
         <p>Has made multiple contributions to the project</p>
     </td>
     <td>
-        <p>Member of the GitHub cloudfoundry (and possibly the cloudfoundry-incubator) orgs</p>
-        <p>Member of the Cloud Foundry Slack workspace</p>
+        <p>Member of a CFF GitHub Organization</p>
     </td>
-    <td>GitHub Organization</td>
+    <td>Per GitHub Organization</td>
```

Comment:
Technically, "contributors.yml" needs more structure if we want to mange contributors per per org.
Maybe you can add this to the RFC draft similar to "cloudfoundry.yml -> orgs.yml".

Alternative would be the scope "All CFF GitHub Orgs". Not sure if we will have many contributors that are active in multiple GitHub orgs and what is the impact on cost and org automation runtime.

Reply:
I would prefer the option that contributors are members in all CFF orgs which are part of the automation because it is easier to explain to new contributors, feels like a community (different CF orgs are just a detail) and the automation implementation should be simpler.